src/pages/inquiry/list/index.tsx

```diff
@@ -0,0 +1,177 @@
+			headerStyle: { width: "90px" },
+		},
+		{ dataField: "name", text: "Title", headerStyle: { width: "200px" } },
+		{ dataField: "email", text: "Email", headerStyle: { width: "200px" } },
+		{ dataField: "message", text: "Message", headerStyle: { width: "200px" } },
+	];
+
+	// Table action buttons
+	const actionButtonFormatter = (row: any) => {
+		return (
+			<span className="dropdown show">
+				<span className="dropdown">
+					<span className="btn shadow-none btn-sm" data-mdb-toggle="dropdown">
+						<i className="fas fa-ellipsis-h"></i>
+					</span>
+					<div className="dropdown-menu dropdown-menu-right">
+						<button className="dropdown-item" onClick={(e) => handleSetDeleteEvent(e, row._id)}>
+							<i className="far fa-trash-alt" /> Delete
+						</button>
+					</div>
+				</span>
+			</span>
+		);
+	};
+
+	const handleSetDeleteEvent = (event: any, contactUsId: string) => {
+		if (event) {
+			dispatch(setContactUsId(contactUsId));
+			$("#eventDeleteModal").modal("show");
+		}
+	};
+
+	const expandRow = {
+		showExpandColumn: true,
+		expandByColumnOnly: true,
+		onlyOneExpanding: true,
+		expandHeaderColumnRenderer: ({ isAnyExpands }: any) => {
+			if (isAnyExpands) {
+				return <b style={{ cursor: "pointer" }}></b>;
+			}
+			return <b style={{ cursor: "pointer" }}></b>;
+		},
+		expandColumnRenderer: ({ expanded }: any) => {
+			if (expanded) {
+				return (
+					<div style={{ cursor: "pointer", marginTop: "5px" }}>
+						<i className="fas fa-chevron-circle-up"></i>
+					</div>
+				);
+			}
+			return (
+				<div style={{ cursor: "pointer", marginTop: "5px" }}>
+					<i className="fas fa-chevron-circle-down"></i>
+				</div>
+			);
+		},
+		renderer: (row: IContactUs) => (
+			<div>
+				<h5>ContactUs Information</h5>
+				<div className="row">
+					<div className="col-md-2 col-sm-12">
+						<h5 className="row-header">Contact Name</h5>
+					</div>
+					<div className="col-md-10 col-sm-12">
+						<p>Y{row.name}</p>
+					</div>
+					<div className="col-md-2 col-sm-12">
+						<h5 className="row-header">Contact Email</h5>
+					</div>
+					<div className="col-md-10 col-sm-12">
+						<p>{row.email}</p>
+					</div>
+					<div className="col-md-2 col-sm-12">
+						<h5 className="row-header">Contact Message</h5>
+					</div>
+					<div className="col-md-10 col-sm-12">
+						<p>{row.message}</p>
+					</div>
+				</div>
+			</div>
+		),
+	};
+
+	return (
+		<div className="card">
+			<div className="row">
+				<div className="col-6">
+					<h3 className="page-title">Events</h3>
+					<p className="page-description text-muted">Manage all the ContactUs informations</p>
+				</div>
+				<div className="col-6">
+					<div className="d-flex justify-content-end">
+						<button
+							className="btn btn-primary btn-rounded shadow-none"
+							data-mdb-toggle="modal"
+							data-mdb-target="#addEventModal"
+						>
+							<span className="fas fa-plus" />
+							<span className="mx-2">Add ContactUs</span>
```

Comment:
This is copy paste code from the event component. In contact us we are not creating any entries. All the contact us creation will happen on the website end. No need to have `Add Contactus` part in here.